config/server.js

```diff
@@ -0,0 +1,9 @@
+module.exports = ({ env }) => ({
+  host: env('HOST', '0.0.0.0'),
+  port: env.int('PORT', 1337),
+  admin: {
+    auth: {
+      secret: env('ADMIN_JWT_SECRET', '403e65b2a6b2809f61304c168740c672')
```

Comment:
Denne secret bør kanskje ikke være åpen tilgjengelig her siden repo ligger tilgjengelig for alle. Bør heller defineres som en secret i GCP. Men kan evt. gjøres senere hvis det viktigste er å få opp noe nå.

Reply:
Jeg mener den egentlig bør fjernes helt, men den kommer med standard.

Reply:
@valosnah Kanskje vi kan bare la den stå, også genererer vi en egen som legges inn i GCP?

Reply:
Så er denne for utvikling?

Reply:
Ok hvis det ikke er behov for den så kan du fjerne den.

Reply:
Tanken var at den bør folk definere selv i en `.env`. [Se en _litt_ bedre flyt på nettsiden til Strapi](https://strapi.io/documentation/developer-docs/latest/update-migration-guides/migration-guides/migration-guide-3.0.x-to-3.1.x.html#_2-define-the-admin-jwt-token).